modules/nf-core/gatk4/leftalignandtrimvariants/main.nf

```diff
@@ -25,14 +25,14 @@ process GATK4_LEFTALIGNANDTRIMVARIANTS {
     def args             = task.ext.args   ?: ''
     def prefix           = task.ext.prefix ?: "${meta.id}"
     def interval_command = intervals       ? "--intervals $intervals" : ""
-    def avail_mem = 3
+    def avail_mem = 3072
     if (!task.memory) {
         log.info '[GATK LeftAlignAndTrimVariants] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
     } else {
         avail_mem = task.memory.toGiga()
```

Comment:
Need to update this?

Reply:
👍🏻 Done!

Reply:
Sorry, I meant changing the line `avail_mem = task.memory.toGiga()` to the new 0.8 calculation.